watertap/examples/flowsheets/full_treatment_train/analysis/multi_sweep.py

```diff
@@ -244,7 +244,7 @@ def run_analysis(case_num, nx, RO_type, output_directory=None, interp_nan_output
         outputs["Ca Removal"] = m.fs.removal_Ca
         outputs["Mg Removal"] = m.fs.removal_Mg
         outputs["Annual Water Production"] = m.fs.costing.annual_water_production
-        outputs["capital_cost_total"] = m.fs.costing.total_capital_cost
+        outputs["capital_cost_total"] = m.fs.costing.aggregate_capital_cost
```

Comment:
this confuses me (probably because I'm not familiar with watertap norms) - why the key is called `capital_cost_total` while the value is the `aggregate_capital_cost`? I thought `aggregate_capital_cost` is for direct CAPEX?

Reply:
https://github.com/watertap-org/watertap/blob/b54753b47b935e4f7aa99eff2cb7e1840e376efd/watertap/examples/flowsheets/full_treatment_train/analysis/multi_sweep.py#L180

Purely historical reasons, which was way this divergence existed in the first place. I'm hesitant to change the key in the output dictionary, however. The value has been consistent throughout (even though it has had different names).

Reply:
OK, then LGTM, thanks!